src/MessagePack.Experimental/UnsafeBlittableFormatter/UnsafeUnmanagedStructArrayFormatter.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+#pragma warning disable SA1649 // File name should match first type name
+
+namespace MessagePack.Formatters
+{
+    public sealed unsafe class UnsafeUnmanagedStructArrayFormatter<T> : IMessagePackFormatter<T[]?>
```

Comment:
Is there any value to defining this as `Memory<T>` or `Span<T>` or their readonly counterparts instead of requiring an array?

Reply:
`Span<T>` can not be the generic type argument.
I made `Memory<T>`, `ReadOnlyMemory<T>` and `ReadOnlySequence<T>` formatters.